### DISCUSSION TITLE
OMI glTF Working Group Meeting 2022-06-23

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/gltf-extensions#91] [General]
This meeting will be on 6/23/2022 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Agenda:

- [KHR_audio](https://github.com/KhronosGroup/glTF/pull/2137) feedback
- [OMI_collider](https://github.com/omigroup/gltf-extensions/pull/63)
- OMI_id / [OMI_ref](https://github.com/omigroup/gltf-extensions/pull/85)
- End of the meeting show and tell period

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by lyuma** [comment]
KHR_audio: Waiting for consensus. Ant and Robert Plan to work on three-omi. Question about if the PR open in the omi github should be under the OMI name.

**Reply by lyuma** [reply]
OMI_collider: some feedback was posted about cylinder, fit/trimesh, trigger (Area node in godot)

OMI_id, OMI_ref: no updates

**Comment by lyuma** [comment]
Talked with memelotsqui about the AVP house demo.
Subject of Lightmaps came up. Compression artifacts usin jpg. Png and ktx2 (basisu) were also used
Uncompressed textures are too big and run out of VRAM.
Had to use lower resolution assets.

We discussed bc6h and float16 extensions but these are inactive pull requests (glTF 1954 and 1955) It might be worth exploring basisu in RGBD format as defined here: https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Vendor/EXT_lights_image_based

We discussed animations. Issues discussed: bezier (weight/tangents), sharing animations between objects

**Reply by lyuma** [reply]
We also discussed light probes: probe spherical harmonics coefficients seem in wide use across many engines. Including three.js, unity, godot 4 alpha. (lyuma: tetrahedralization data? Anything else?)
